lib/lambda_ethereum_consensus/store/block_store.ex

```diff
@@ -3,27 +3,49 @@ defmodule LambdaEthereumConsensus.Store.BlockStore do
   Storage and retrieval of blocks.
   """
 
-  @prefix "block"
+  @block_prefix "block"
+  @slothash_prefix "slothash"
 
-  @spec store_block(SszTypes.BeaconBlock.t()) :: any
+  @spec store_block(SszTypes.BeaconBlock.t()) :: :ok
   def store_block(%SszTypes.BeaconBlock{} = block) do
     {:ok, block_root} = Ssz.hash_tree_root(block)
     {:ok, encoded_block} = Ssz.to_ssz(block)
 
-    key = @prefix <> block_root
+    key = get_key(@block_prefix, block_root)
```

Comment:
Nit and optional: now that we have two different prefixes, I'd add a `block_key(root)` and `root_by_slot_key(slot)` functions to wrap around the `get_keys` function.

Reply:
That's a good idea!